### PR TITLE
Filter alerts by category

### PR DESCRIPTION
**Pull request by maxnoe**
This adds a form on top of the alerts table to filter alerts by category and only the shifter alerts are show by default

### CODE REVIEW

**Comment by dneise** [comment]
To review this, what do you propose to do?

Check it out and run it using "docker-compose up"? Or did you install it somewhere for testing, where we could look at it?

**Comment by dneise** [comment]
Ah I see, the question I just asked is actually answered in the README of this PR. nice

File: README.md [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
 
+## Technology stack
 
-# how to start:
+We are using
 
- - clone
- - cd into 
+* twitter bootstrap for responsivness (make it look good on mobile)
+* react (without jsx) for the ui component
```
Comment (dneise) [inline README.md]: what is jsx? This? https://jsx.github.io/

what do you mean by: "the ui component?"

File: README.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+* socketio for the websocket (new alerts pop up without refreshing the page)
 
-# 
-Build the container:
+React using pure javascript avoids a more complicated compilation step but 
+results in more boilerplate code.
```
Comment (dneise) [inline README.md:13]: This seems to be the answer to some question like: "Why is there so much boilerplate?" ... I am not advanced enough to have even gotten to this question. Can you elaborate a bit, where this boilerplate code is?

File: README.md [hunk 3]
```diff
@@ -24,16 +32,11 @@ $ docker run --restart=always \
 	--name=shifthelper_mysql \
 	mysql
 ```
```
Comment (dneise) [inline README.md:37]: this is supposed to be a test mysql DB right? So is it necessary to do all these security measures? 

Oh, Is the mysql inside this docker container by default running on port 3306 and thus open to the outside world? Then I get this

File: README.md [hunk 4]
```diff
@@ -24,16 +32,11 @@ $ docker run --restart=always \
...
-Start the shifthelper container
+
+Adapt the database settings in the config file to reflect your local db setup,
+then run
```
Comment (dneise) [inline README.md:40]: you mean the MYSQL_USER and MYSQL_PASSWORD from above right? 
So alternatively one could use the values from the webservice.json in the `docker run mysql` call and avoid touching the config file, yes?

**Review by dneise — CHANGES_REQUESTED** [review]
I would approve, but have not yet had time to check it out and look at it.

File: README.md (hunk 1 above)
Comment (maxnoe) [reply README.md]: See https://reactjs.org/docs/introducing-jsx.html

UI is basically only the alert table at the moment

**Review by dneise — APPROVED** [review]

**Pull request merged**